modules/editable-layers/package.json

```diff
@@ -71,15 +71,15 @@
     "viewport-mercator-project": ">=6.2.3"
   },
   "peerDependencies": {
-    "@deck.gl-community/layers": "^9.1.0-beta.2",
-    "@deck.gl/core": "^9.1.12",
-    "@deck.gl/extensions": "^9.1.0",
-    "@deck.gl/geo-layers": "^9.1.0",
-    "@deck.gl/layers": "^9.1.0",
-    "@deck.gl/mesh-layers": "^9.1.0",
-    "@luma.gl/constants": "^9.1.0",
-    "@luma.gl/core": "^9.1.0",
-    "@luma.gl/engine": "^9.1.0",
+    "@deck.gl-community/layers": "~9.1.0-beta.2",
```

Comment:
Is this correct? Is `@deck.gl-community/layers` versioned in the same way, where minor versions have breaking changes?